app/views/content/_content_row.html.erb

```diff
@@ -1,5 +1,5 @@
 <tr>
-  <td><%= item.title %>
+  <td><%= link_to item.title, "/metrics#{item.base_path}" %>
```

Comment:
If you add the `:as` clause to `routes.rb`:

```ruby
  get '/metrics/*base_path', to: 'metrics#show', as: :metrics
```

You can use the helper for the route:

```ruby
  metrics_path('the/base/path')
```